mcs/class/System.Windows.Forms/System.Windows.Forms/GridEntry.cs

```diff
@@ -827,7 +819,7 @@ private string[] GetMergedPropertyNames (object [] objects)
 			string[] propertyNames = new string [intersection.Count];
 			for (int i=0; i < intersection.Count; i++)
 				propertyNames[i] = ((PropertyDescriptor)intersection[i]).Name;
-				
+
```

Comment:
Reduce line damage. Revert that change.

Reply:
same comment, as https://github.com/mono/mono/pull/11618/commits/32f310893f277ab4d1fca1bc69744c0ac30d1969#r234181420